baseplate/lib/live_data/zookeeper.py

```diff
@@ -83,6 +90,6 @@ def zookeeper_client_from_config(
             backoff=2,  # exponential backoff
             max_jitter=1,  # maximum amount to jitter sleeptimes
             max_delay=60,  # never wait longer than this
-            sleep_func=gevent.sleep,
+            sleep_func=time.sleep,
```

Comment:
We could just rely on the default value, but passing in time.sleep makes mocking much easier.

Reply:
I would agree, if this isn't different from the defaults it doesn't make much sense to pass it.

Reply:
:no_bell: Can you add a comment to this effect here?

Reply:
@nsheaps  I can do that, but the patching gets much nastier because default values are not easily patched 🤢 I just pushed a commit with that change so let me know what you think

Reply:
if passing time.sleep makes this all easier, feel free to pass it. If we want to be sticklers about it, then max_delayu is also defaulting to 60, and backoff to 2 ( [source](https://github.com/python-zk/kazoo/blob/master/kazoo/retry.py#L45) )